themes/cardinal_service/lib/components/opportunity-row/opportunity_row.html.twig

```diff
@@ -1,24 +1,77 @@
-<div{{ attributes.addClass('flex-container') }}>
-  <div class="flex-9-of-12">
+<<<<<<< HEAD
+<div class="su-opportunity-result">
+  <div{{ attributes.addClass('flex-container su-card__contents') }}>
```

Comment:
`attributes` is intended for the outer most container. can you move this back and apply necessary classes needed there. or just switch where the classes sit.